### ISSUE TITLE
Implement auto-pause feature

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#136]
minikube is adding an "auto-pause" feature to pause all containers if they haven't received an external request in X minutes.

This is implemented by a simple proxy, that runs a timer, and pauses the cluster when the timer expires.

When a request comes in, the timer is reset. And if the cluster is already paused, it will be unpaused before the request is passed on.

**Comment by gaktive** [comment]
We've left minikube and have other tickets tied to something along the line of pausing.

**Comment by jandubois** [comment]
> We've left minikube and have other tickets tied to something along the line of pausing.

I don't think this should be closed. It has nothing to do with using minikube or not; we just want to copy the specific feature; or something that achieves a similar effect of preserving battery when not in use.

I don't see any other tickets talking about pausing containers. Please include issue numbers if you think they supersede the current issue!

**Comment by gaktive** [comment]
#381 was the main one, though it reads differently as I look through the backlog. The other (#1274) were tied more to sleeping vs. pausing so if there's merit in using this instead, I'm ok with that.
